mmv1/products/tags/api.yaml

```diff
@@ -149,3 +149,32 @@ objects:
 
           A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
         output: true
+  - !ruby/object:Api::Resource
+    name: 'TagBinding'
+    min_version: beta
+    base_url: tagBindings
+    self_link: "tagBindings/?parent={{parent}}&pageSize=300"
```

Comment:
Have we asked the API team to support Get? This isn't going to scale past 300 bindings in an organization, and we could consider it a blocker for Terraform support.

Reply:
I can ask them about this.

Reply:
b/184380816

I suspect that there might be a limit of 300 TagBindings for a specific parent (which is the resource being tagged rather than the organization) since there is a limit of 300 TagValues per TagKey, but it's not documented one way or another. I think that it's unlikely someone will immediately try to make over 300 tags on a specific parent using terraform so we might be okay to release it like this for now?